Pcap++/header/PcapLiveDevice.h

```diff
@@ -186,6 +186,38 @@ namespace pcpp
 			PCPP_OUT
 		};
 
+		/**
+		 * Set which source provides timestamps associated to each captured packet
+		 * (you can read more here: <https://www.tcpdump.org/manpages/pcap-tstamp.7.html>)
+		 */
+		enum TimestampProvider
```

Comment:
Can we use `enum class` here? we are trying to move away from plain enums where possible and its a pain to migrate after it has already been set.

Reply:
done